### ISSUE TITLE
picamera required dependeny

### ISSUE THREAD

**Issue by brendenpelkie** [machineagency/science_jubilee#38]
The picamera package is listed as a dependency in setup.py, making science_jubilee uninstallable on a non-pi system (or at least my WSL ubuntu instance). 

Proposed fix:
1. remove picamera from base install dependencies
2. add  steps to check install/manage install of picamera to camera tools that directly run on a pi

**Comment by bsubbaraman** [comment]
Good catch-- currently it installs fine on mac or windows, but [the conditional check in setup.py](https://github.com/machineagency/science_jubilee/blob/39ba333df7cbecc58c52599d3cd4dbf56f385dc7/setup.py#L26C9-L26C9) is currently if running on linux; I would propose changing this to check the os_name to only install on Raspberry Pi!

**Comment by brendenpelkie** [comment]
That makes sense too. That's cool I didn't know you could do conditional installs like that!

**Comment by brendenpelkie** [comment]
Addressed by #41. Checks for pi camera by checking for aarch64 machine instead of linux platform.

**Comment by bsubbaraman** [comment]
Reopening because I just tried a fresh install on my pi 3 which isn't aarch64 so picamera doesn't install! I'll submit a pull request with a fix for this one
